lib/adapters/tick.ts

```diff
@@ -84,8 +85,8 @@ export class TickAdapter implements TargetAdapter<TickCredentials> {
       url: `${this.subURL}/entries.json`,
       headers: this.authHeaders,
       params: {
-        start_date: from.toISOString().split("T")[0],
-        end_date: to.toISOString().split("T")[0],
+        start_date: localizedPlainDateToUTC(from).toISOString().split("T")[0],
+        end_date: localizedPlainDateToUTC(to).toISOString().split("T")[0],
```

Comment:
please extract this into a new function that could be used like:
 ```suggestion
        start_date: localPlainDateToString(from),
        end_date: localPlainDateToString(to),
```